R/checker.R

```diff
@@ -286,6 +290,18 @@ checker <- R6::R6Class(
 
     pop_process = function(name) {
       private$active[[name]] <- NULL
+    },
+    # With these approach we deviate slightly from always prioritizing check
+    # tasks across the entire graph in favor of prioritizing them on the given
+    # layer. We do that due to significant performance gain in graph searching.
+    get_next_node = function(force = FALSE) {
+      ready_nodes <- task_graph_which_ready(self$graph)
+      if (length(ready_nodes) == 0 | force) {
```

Comment:
```suggestion
      if (length(ready_nodes) == 0 || force) {
```

Minor, but `lintr` will yell at you for this. Probably should run `lintr` against this PR if you aren't already.

Reply:
I did and surprisingly lint yells about single `&` but not about single `|`, which is pretty bizarre :D Thanks for catching that